src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -249,6 +249,10 @@ <h2><span jhiTranslate="artemisApp.exercise.details">Exercise Details</span></h2
             </div>
             <dl class="row-md jh-entity-details">
                 <jhi-exercise-details [exercise]="programmingExercise"></jhi-exercise-details>
+                <dt><span jhiTranslate="artemisApp.exercise.feedbackSuggestionsEnabled">Athena feedback suggestions enabled</span></dt>
```

Comment:
I think I would prefer `Enable feedback suggestions from Athena` but leave it up to you

Reply:
I also like that better & changed it